### ISSUE TITLE
How to return arbitrary result?

### ISSUE THREAD

**Issue by TvoroG** [tarantool/nginx_upstream_module#133]
Thanks a lot for the module!

I want to wrap json object into callback: 
```
callback({"foo": "bar"})
```

I don't know how to do that. I've tried to return result as simple string from tarantool with following nginx config:

```
tnt_pass router;
tnt_method 'foo_method';
tnt_pure_result on;
tnt_multireturn_skip_count 2;
```

But module returns it wrapped into double quotes
```
"callback({\"foo\": \"bar\"})"
```

**Comment by snoopcatt** [comment]
Please, show your `foo_method` code.

**Comment by TvoroG** [comment]
It simply returns a string
```lua
function foo_method()
    return 'callback({"foo": "bar"})'
end
```

**Comment by snoopcatt** [comment]
And what do you expect?

**Comment by Totktonada** [comment]
Try something like this: https://github.com/tarantool/nginx_upstream_module/tree/82355b8a1276ad584308f3ee4cc6e6129ecb302f#http-headers-and-status

**Comment by TvoroG** [comment]
I'm expecting `callback({\"foo\": \"bar\"})` but it returns `"callback({\"foo\": \"bar\"})"`.

> Try something like this: https://github.com/tarantool/nginx_upstream_module/tree/82355b8a1276ad584308f3ee4cc6e6129ecb302f#http-headers-and-status

Thanks, but i was hoping to do it without lua on nginx.

**Comment by Totktonada** [comment]
I agree that it is not convenient to add extra logic on nginx side just to implement an API you need. AFAIU the module was initially made with RPC in a mind, that is why we need to preprocess-postprocess a request and a response to implement, say, restful API. However it would be good to change it in the future.

**Comment by Totktonada** [comment]
Hope we should not close the issue, but rather add an option that will do something like the Lua code that I linked above.
